index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel = "stylesheet"  type = "text/css" href = "style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <title></title>
+  </head>
+  <body>
+    <header>
+            <img src="img/menu_bar.png" width="25" height="25" style="padding:17px;" />
+            <img src="img/youtube.png" width="100" height="64" />
+            <form class="search-form">
+              <input type="search" placeholder="search" >
+              <button type="submit" class="search-button">
+                <img src="img/search-icon.png"/>
+              </button>
+            </form>
+            <img src="img/account-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/bell-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/message-icon.png" width="40" height="25" style="padding:17px;float:right;" />
+            <img src="img/app-menu-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/camera-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+
+    </header>
+    <div id="margine">
```

Comment:
You are using percentages for the `margin-left` and `margin-right` properties and the result is that if the window width changes, the alignment breaks. Try it out, change the browser width to half the screen and see what happens.